crates/worker/src/partition/types.rs

```diff
@@ -90,7 +89,7 @@ pub fn create_response_message(
                     caller_context,
                 }
                 .encode_to_vec(),
-                Source::Service(callee.clone()),
+                Source::Internal,
```

Comment:
By taking this shortcut, we can preserve the `Source::Service`. This only affects internal callbacks (which we'll stop using anyway now that i'm reimplementing idempotency)